stage-item/Gemfile

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+
+# gem "rails"
+
+gem "marc", "~> 1.2"
+gem "faraday", "~> 2.7"
+gem "faraday-follow_redirects"
+gem "ht-pairtree", git: "../../ht-pairtree"
```

Comment:
needs to be updated to point to https://github.com/hathitrust/ht-pairtree/ after that's merged